agents/schemas/profile.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "agents/Profile",
+  "title": "Profile",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "integer",
+      "description": "Id referencing profile"
+    },
+    "agentId": {
+      "type": "integer",
+      "description": "Id referencing agent"
+    },
+    "name": {
+      "type": "string",
+      "faker": "name.findName",
+      "description": "An agent name"
+    },
+    "description": {
+      "type": "string",
+      "description": "Agent biography"
+    },
+    "avatar" : {
+      "type": "string",
+      "faker": "internet.avatar",
+      "format": "uri",
+      "description": "Agent avatar image link"
+    }
+  },
+  "required": [
+    "id",
+    "agentId",
+    "name"
```

Comment:
i'm not sure if `name` is required. or at least, at the moment i think we create "empty" profiles when we create a group agent to be updated later.